src/Pagination/index.jsx

```diff
@@ -371,7 +326,7 @@ Pagination.propTypes = {
   /** specifies the total number of pages in the `Pagination` component. */
   pageCount: PropTypes.number.isRequired,
   /** specifies the `aria-label` for the `<nav>` element that wraps the pagination button list. */
-  paginationLabel: PropTypes.oneOfType([PropTypes.string, PropTypes.element]).isRequired,
+  paginationLabel: PropTypes.string.isRequired,
```

Comment:
I noted that this label also specified it could be an element, but that it stuffs the value directly into an `aria-label` attribute.  There's no way that could be anything other than a string, so I removed the erroneous prop type.